inji-web/src/components/Home/HomeFeatures.tsx

```diff
@@ -17,39 +17,57 @@ export const HomeFeatures: React.FC = () => {
   const handlePrev = () => {
     setCurrentFeature((prev) => (prev - 1 + totalFeatures) % totalFeatures);
   };
- 
+
   return (
-    <div className="flex justify-center items-center flex-col">
-      <div className="font-semibold text-3xl m-5">{t("Features.heading")}</div>
-      <div className="font-extralight text-center">{t("Features.description1")}</div>
-      <div className="font-extralight text-center mb-10">{t("Features.description2")}</div>
+    <div data-testid="HomeFeatures-Container" className="flex justify-center items-center flex-col">
+      <div data-testid="HomeFeatures-Heading" className="font-semibold text-3xl m-5">
+        {t("Features.heading")}
+      </div>
+      <div data-testid="HomeFeatures-Description1" className="font-extralight text-center">
+        {t("Features.description1")}
+      </div>
+      <div data-testid="HomeFeatures-Description2" className="font-extralight text-center mb-10">
+        {t("Features.description2")}
+      </div>
 
-      <img className="block sm:hidden" src={require("../../assets/Content.png")} alt="Inji Web Mobile Preview" />
-      <img className="hidden sm:block" src={require("../../assets/InjiWebPreview.png")} alt="Inji Web Preview" />
+      <img data-testid="HomeFeatures-MobileImage" className="block sm:hidden" src={require("../../assets/Content.png")} alt="Inji Web Mobile Preview" />
+      <img data-testid="HomeFeatures-DesktopImage" className="hidden sm:block" src={require("../../assets/InjiWebPreview.png")} alt="Inji Web Preview" />
 
-      <div className="flex flex-wrap gap-8 container mx-auto pb-3 md:pb-20 justify-center">
-        <div className="hidden md:flex flex-wrap gap-8 justify-center">
-            {[1,2,3,4,5].map( itemNo => <HomeFeatureItem itemno={itemNo} /> )}
+      <div data-testid="HomeFeatures-ItemsContainer" className="flex flex-wrap gap-8 container mx-auto pb-3 md:pb-20 justify-center">
+        <div data-testid="HomeFeatures-Items" className="hidden md:flex flex-wrap gap-8 justify-center">
+          {[1, 2, 3, 4, 5].map((itemNo) => (
+            <HomeFeatureItem key={itemNo} itemno={itemNo} />
+          ))}
         </div>
-        <div className="sm:hidden mx-9 my-6 h-[32rem] w-full flex justify-center">
-            <HomeFeatureItem itemno={currentFeature + 1} />
-          </div>
+        <div data-testid="HomeFeatures-MobileItem" className="sm:hidden mx-9 my-6 h-[32rem] w-full flex justify-center">
```

Comment:
Dont need Wrapper here. add it inside HomeFeatureItem

Reply:
Added already